#!/usr/bin/env python
# coding: utf-8

# Autor : Aclelino Dami√£o <br>
# E-mail: aclelino@gmail.com <br>
# GitHub: https://github.com/Aclelino<br>
# 
# Python 3.8.8 <br>
# 
# '''<br>
# Calculadora de imc <p>
# c√°lculo simples que permite medir se algu√©m est√° ou n√£o com o peso ideal. Ele aponta se o peso est√° adequado ou se est√° abaixo ou acima
# <br>
# '''

# In[1]:



from PyQt5 import uic,QtWidgets


# In[2]:


def acao():
    peso =float(windows.lineKg.text())
    altura = float(windows.lineMt.text())
    
    print(peso)
    imc = peso / altura ** 2
    #imc = str(imc)
    resultado = windows.label_Resultado
    inf = windows.label_Inf
    
    inf.setText('O seu IMC e igual a {:.2f}\n'.format(imc))

    if imc < 16:
        resultado.setText("Magreza grau IIIüòê")
    elif imc > 16.0 and imc < 16.9 :
        resultado.setText("Magreza grau IIü§®")
    elif imc > 17.0 and imc < 18.4 : 
        resultado.setText("Magreza grau Iü§î")
    elif imc > 18.5 and imc < 24.9 :
        resultado.setText("Adequado üòä")
    elif imc > 25.0 and imc < 29.9:
        resultado.setText("Pr√©-obesoüòè")
    elif imc > 30.0 and imc < 34.9 :
        resultado.setText("Obesidade grau Iüò£")
    elif imc > 35.0 and imc < 39.9:
        resultado.setText("Obesidade grau IIüò•")
    elif imc >=40.0:
        resultado.setText("Obesidade grau IIIüò´")



# In[3]:


app =QtWidgets.QApplication([])
windows = uic.loadUi('imc.ui')
windows.pushButton.clicked.connect(acao)
windows.show()
app.exec_()


# In[ ]:




